Merge 4.43 code freeze to develop

oguzkocer opened the pull request.

oguzkocer committed:
Bump version number

oguzkocer committed:
Update draft release notes for 4.43.

oguzkocer committed:
Release Notes: add new section for next version (4.44)

oguzkocer committed:
Update strings for localization

oguzkocer committed:
Update release notes for 4.43

oguzkocer committed:
Update metadata strings

AliSoftware commented on Simplenote/Resources/AppStoreStrings.pot:
```diff
@@ -15,7 +15,8 @@ msgstr ""
 #. translators: Subtitle to be displayed below the application name in the Apple App Store. Limit to 30 characters including spaces and commas!
 msgctxt "app_store_subtitle"
-msgid "Simple notes, todos and memos"
+msgid "Simple notes, todos and memos
+"
```
This bug from our toolkit that generates the `.pot` strikes again 😒 

```suggestion
msgid "Simple notes, todos and memos"
```

AliSoftware commented on Simplenote/Resources/AppStoreStrings.pot:
```diff
@@ -63,12 +64,14 @@ msgstr ""
 #. Delimit with an English comma between each keyword. Limit to 100 characters including spaces and commas.
 msgctxt "app_store_keywords"
-msgid "notes,text,planner,journal,diary,sync,to-do,cloud,notebook,simple,notepad,tag,todo,writing,memo"
+msgid "notes,text,planner,journal,diary,sync,to-do,cloud,notebook,simple,notepad,tag,todo,writing,memo
+"
```
ditto here

```suggestion
msgid "notes,text,planner,journal,diary,sync,to-do,cloud,notebook,simple,notepad,tag,todo,writing,memo"
```

mokagio replied:
I took a stab at fixing this issue here https://github.com/wordpress-mobile/release-toolkit/pull/297.

AliSoftware commented:
PS: As a workaround for next time, I wonder if, until we find some time to properly fix it in the toolkit, removing the trailing `\n` from the corresponding `…/source/{keywords,subtitle}.txt` files could help avoid this bug for next releases in the meantime?

mokagio committed:
Manually fix incorrect new lines in .pot file

We are working on a proper fix here:
https://github.com/wordpress-mobile/release-toolkit/pull/297

mokagio reviewed (commented):
- [x] New version header in `RELEASE-NOTE.txt` 
- [x] Version update in `.xcconfig`, `Deliverfile`, and `download_metadata.swift` 
- [x] `Localizable.strings` updated 
- [x] `release_notes.txt` updated with notes from `RELEASE-NOTE.txt` for current version 
- [x] `AppStoreStrings.pot` updated accordingly to new version and release notes

mokagio commented:
I pushed 57f7bb1 to address the `.pot` issue and reviewed the PR.

I'm going to dismiss @AliSoftware 's request for changes to let the auto-merge kick in so the new strings can get to GlotPress.
